docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -245,23 +252,23 @@ Our project has new WCF client classes now (in Reference.cs), but it also still
 
 01. If on the other hand some code churn in the existing .NET Framework project is acceptable, you can remove *BeanTrader.cs* all together. Because the new WCF client is built for .NET Standard, it will work in both .NET Core and .NET Framework scenarios. If you are building for .NET Framework in addition to .NET Core (either by multi-targeting or by having two csproj files), you can use this new Reference.cs file for both targets. This approach has the advantage that the code won't need to bifurcate to support two different WCF clients - the same code will be used everywhere. The drawback, of course, is that it involves changing the (presumably stable) .NET Framework project.
 
-In the case of the Bean Trader sample, we can make small changes to the original project if it makes migration easier, so follow these steps to reconcile WCF client usage:
+In the case of the Bean Trader sample, you can make small changes to the original project if it makes migration easier, so follow these steps to reconcile WCF client usage:
 
 01. Add the new Reference.cs file to the .NET Framework *BeanTraderClient.csproj* project using the 'Add existing item' context menu from the solution explorer. Be sure to add 'as link' so that the same file is used by both projects (as opposed to copying the C# file). If you are building for both .NET Core and .NET Framework with a single csproj (via multi-targeting) then, of course, this step isn't necessary.
 
 01. Delete *BeanTrader.cs*.
 
 01. The new WCF client is similar to the old one, but a number of namespaces in the generated code are different. Because of this, it is necessary to update the project so that WCF client types are used from **BeanTrader.Service** (or whatever namespace name you chose) instead of BeanTrader.Model or without a namespace. Building BeanTraderClient.Core.csproj will help to identify where these changes need to be made. Fixes will be needed both in C# and in XAML source files.
```

Comment:
```suggestion
01. The new WCF client is similar to the old one, but a number of namespaces in the generated code are different. Because of this, it is necessary to update the project so that WCF client types are used from **BeanTrader.Service** (or whatever namespace name you chose) instead of BeanTrader.Model or without a namespace. Building *BeanTraderClient.Core.csproj* will help to identify where these changes need to be made. Fixes will be needed both in C# and in XAML source files.
```